modules/Campaigns/WizardMarketing.php

```diff
@@ -271,8 +271,14 @@
 }
 
 $outboundEmailAccountLabels = array();
-foreach($outboundEmailAccounts = BeanFactory::getBean('OutboundEmailAccounts')->get_full_list() as $outboundEmailAccount) {
-    $outboundEmailLabels[$outboundEmailAccount->id] = $outboundEmailAccount->name;
+$outboundEmailLabels = array();
+$outboundEmailAccounts = BeanFactory::getBean('OutboundEmailAccounts')->get_full_list();
+if ($outboundEmailAccounts) {
+    foreach ($outboundEmailAccounts as $outboundEmailAccount) {
+        $outboundEmailLabels[$outboundEmailAccount->id] = $outboundEmailAccount->name;
+    }
+} else {
+    $GLOBALS['log']->warn('No any outbound email account');
```

Comment:
Can we change the message to something like "There are no outbound email accounts available. Please ensure that the email settings are configured correctly"?